src/spec/page-objects/settings-page.js

```diff
@@ -8,6 +8,7 @@ class SettingsPage {
     this._domainInput = $('.domain--default-container input');
     this._domainInputButton = $('.domain--default-container button');
     this._domainsSetDefaultButtonList = element.all(by.css('.domain--default-action'));
+    this._domainDeleteButton = element.all(by.css('.domain--erase-button'));
```

Comment:
What do you think about to pluralize it? It could represent more than one, right?